x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/persistence_local_scheduled_task_creation.json

```diff
@@ -51,5 +51,5 @@
     }
   ],
   "type": "eql",
-  "version": 11
+  "version": 12
```

Comment:
Version already bumped in previous `8.2 Rules` PR: https://github.com/elastic/kibana/pull/129526/files#diff-548404a51453422eb3260dd0bfe31d2fbcb4a451d2629a598aea5df3991ccab5R54